### ISSUE TITLE
Introduce page with release info

### ISSUE THREAD

**Issue by keunes** [nextcloud/nextcloud.com#504]
A few users (including me) would like some more information about the rolling out of updates in the different channels. I myself am in the 'stable' channel and have not yet received an update notification. In order for people to check and avoid having to have a forum thread with every major update, it would be nice to:

* have a page (say, www.nextcloud.com/update) where updating information is published
* there, make available:
  * the versions available in the different channels (from [shoeper](https://help.nextcloud.com/t/nextcloud-12-release-channel/14372/16))
  * the roll-out percentage with each update ("Currently, this update is offered to xx % of the users")
* and possibly also:
  * a notification if roll-out is on hold due to a found bug
  * a rough estimation for the roll-out, such as amount of batches and planned/expected time ("There will be xx stages over the next yy weeks") (from [Thor](https://help.nextcloud.com/t/nextcloud-12-release-channel/14372/10))
  * a graph showing downloads a day, updated, downloading and waiting users in the queue... (from [Jove](https://help.nextcloud.com/t/nextcloud-12-release-channel/14372/18))
* link to that page from the update section in nextcloud server (from [janvlug](https://help.nextcloud.com/t/nextcloud-12-release-channel/14372/3))

**Comment by MariusBluem** [comment]
We should improve https://nextcloud.com/release-channels for solving that ;)

**Comment by jospoortvliet** [comment]
Yeah, very good idea. @keunes any chance you have a look at the release channels page and perhaps work some variables magic in there?

**Comment by keunes** [comment]
Hi @jospoortvliet 
I'd be happy to but I'm merely a translator for Nextcloud, and someone using Nextcloud for home-sharing with a wish for more info on the upgrade process 🙂. In other words: my HTML and CSS skills are limited to, lets say, well, they're limited ^^ (let alone PHP variables).

I'm afraid I don't have the time to learn to do the work required, at the moment. So if anyone else would be happy to take that on, that'd be great :)

**Comment by jospoortvliet** [comment]
Ok, can do. With more thinking I believe the problem will be to keep it up to date: if we do that manually it will go outdated all the time :(

Would be best if we could get the info directly from the updater server somehow. But that'd be way beyond my PHP fu...

**Comment by janvlug** [comment]
Did you know about the existence of this page:
https://github.com/nextcloud/server/wiki/Maintenance-and-Release-Schedule

**Comment by Espina2** [comment]
@jospoortvliet if you have an API or a json file with all the info, its not that difficult to do. :)

**Comment by jospoortvliet** [comment]
@Espina2 hmmmmmm, we don't have it right now I think, though of course the update server can be queried as that is how the nextcloud servers know if there's a new version ;-) Not sure how complicated it is...

**Comment by Bentong021** [comment]
https://github.com/nextcloud/updater_server/issues/68

**Comment by danger89** [comment]
I like this idea very much and especially since 12 is not fully rolled out, it's confusing for me, since the download page says that 12.0.2 is stable, but the built-in updater won't do anything. By giving more information about the status helps me ( and others)  to understand

**Comment by hairydogltd** [comment]
As far as I can make out, this whole issue came from someone asking about what was in each channel. As far as I can see, lots of people have commeneted, but the initial question has not been answered clearly enough for me to understand. 
This is my question. On the Admin page, I can choose which channel to use. 
I am given four options:
stable
daily
beta
production

Now, it seems to me that daily is the latest and flakiest, beta is code that is neraly ready, but what about the other two? 

Is 'production' more or less stable than 'stable'? 

I guess that any code that is not stable can't be considered for production, so I'd expect that 'production' is 'super stable'. Is that correct?

A simple explanation of what these channel names represent would clarify things for me. 

I'm not asking which releases are in each channel right now, nor am I suggesting a discussion of how stable production code needs to be. I just want to know the order of "stability ranking" for these four channels.
